14_1_table.js

```diff
@@ -0,0 +1,27 @@
+const MOUNTAINS = [
+    { name: 'Kilimanjaro', height: 5895, place: 'Tanzania' },
+    { name: 'Everest', height: 8848, place: 'Nepal' },
+    { name: 'Mount Fuji', height: 3776, place: 'Japan' },
+    { name: 'Vaalserberg', height: 323, place: 'Netherlands' },
+    { name: 'Denali', height: 6168, place: 'United States' },
+    { name: 'Popocatepetl', height: 5465, place: 'Mexico' },
+    { name: 'Mont Blanc', height: 4808, place: 'Italy/France' },
+];
+
+const div = document.getElementById('mountains');
+const table = document.createElement('table');
+
+div.appendChild(table);
+table.innerHTML += `<tr>
+<th>${Object.keys(MOUNTAINS[0])[0]}</th>
```

Comment:
I'd move `Object.keys(MOUNTAINS[0])` to separate variable and then use `forEach` to generate that table